.pre-commit-config.yaml

```diff
@@ -1,5 +1,49 @@
+---
+exclude: .crt$
 repos:
```

Comment:
I'm not using the `mirsg-hooks` because the combination of `ansible-lint` and `yamllint` was almost impossible to configure. The default config for `yamllint` is too restrictive - the main problem being the max line length of 80 chars. One option would be to add a `.yamllint` config file in this repo. While this seems like it should work, because pre-commit is running `yamllint` in isolation a config file would need to [extend the default](https://yamllint.readthedocs.io/en/stable/configuration.html#extending-the-default-configuration) and this doesn't work for `ansible-lint` which combine this config with its own `yamllint` config. The only other option is selectively disabling `yamllint` on whole files, or lines in files, which would mean many inline `yamllint disable` comments.

It is worth noting that `ansible-lint` sets the line length to `160` instead of the default of `80` - https://github.com/ansible/ansible-lint/blob/7849f27453fe1d9c29a43c71cff8f5b666bd43b6/src/ansiblelint/yaml_utils.py#L49-L70

Reply:
Reverted to `mirsg-hooks` after relaxing of `yamllint` rules.